portfolio/src/main/webapp/style.css

```diff
@@ -143,7 +163,7 @@ h5 {
   background-color: white;
   border-bottom-right-radius: 15px;
   border-top-right-radius: 15px;
-  box-shadow: 0 0 1pt .5pt #d5d1d1;
+  box-shadow: 0 0 1pt 0.5pt #d5d1d1;
```

Comment:
Don't need the leading 0 in 0.5

https://google.github.io/styleguide/htmlcssguide.html#Leading_0s

Reply:
when i tried deleting these the shadows didn't work correctly